yt_dlp/extractor/youtube.py

```diff
@@ -533,6 +539,74 @@ def _generate_api_headers(self, ytcfg=None, identity_token=None, account_syncid=
             headers['X-Origin'] = origin
         return headers
 
+    @staticmethod
+    def _build_api_continuation_query(continuation, ctp=None):
+        query = {
+            'continuation': continuation
+        }
+        # TODO: clickTrackingParams appears to be contained within the context
+        # which is extracted from ytcfg (and is different from one extracted with continuation)
```

Comment:
does this cause any issues?

Reply:
Doesn't appear to at the moment, but might be good to keep in mind.

Reply:
> but might be good to keep in mind.

Then why'd you remove the comment? 😕

Reply:
good question. I added a comment back